File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.java
class Solution {
    public int solution(int[] num_list) {
        
        int evenSum = 0;
        int oddSum = 0;
        
        for(int i=0; i < num_list.length; i++){
            if(i%2==0){
                evenSum += num_list[i];            
            }
            else{
                oddSum += num_list[i]; 
            }
        }
        
        if(evenSum > oddSum) {
            return evenSum;
        }
        return oddSum;
    }
}

File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.py
def solution(num_list):
    #인덱스 짝수 = 홀수 번째 원소
    #인덱스 홀수 = 짝수 번째 원소
    #합 비교
    evensum = 0
    oddsum = 0
    
    for i,num in enumerate(num_list):
        if i%2==0:
            evensum += num
        else:
            oddsum += num
        
    if evensum > oddsum:
        return evensum
    else:
        return oddsum